packages/plugins/RSS3/src/SNSAdaptor/index.tsx

```diff
@@ -26,18 +20,11 @@ const sns: Plugin.SNSAdaptor.Definition = {
             label: 'Donations',
             priority: 1,
             UI: {
-                TabContent: ({ socialAddressList = [], identity }) => {
-                    return (
-                        <TabCard
-                            socialAddressList={socialAddressList}
-                            type={TabCardType.Donation}
-                            persona={identity?.publicKey}
-                        />
-                    )
+                TabContent: ({ socialAddress, persona }) => {
+                    return <TabCard socialAddress={socialAddress} type={TabCardType.Donation} persona={persona} />
```

Comment:
1. We cannot change it from a list to a single item. Some plugins may need a list.
2. We can read the persona public key from the `SocialIdentity`.

Reply:
1. All plugins under profile tabs will receive only one address now
2. done